oneview_redfish_toolkit/tests/blueprints/test_create_computer_system.py

```diff
@@ -264,7 +267,10 @@ def test_create_system_when_request_content_has_not_a_valid_network(
         ]
 
         data_to_send = copy.copy(self.data_to_create_system)
-        data_to_send["Id"] = "123456789"
+        # Zone id = SPT id + Encl id
```

Comment:
I think this comment is repeated because on the top of test you already put the Zone id composition.